src/audio/copier/copier_dai.c

```diff
@@ -182,50 +255,19 @@ int copier_dai_create(struct comp_dev *dev, struct copier_data *cd,
 		break;
 	case ipc4_alh_link_output_class:
 	case ipc4_alh_link_input_class:
+#if defined(CONFIG_ACE_VERSION_2_0)
+		dai.type = SOF_DAI_INTEL_HDA;
+		dai.is_config_blob = true;
+		type = ipc4_gtw_link;
+#else
 		dai.type = SOF_DAI_INTEL_ALH;
 		dai.is_config_blob = true;
 		type = ipc4_gtw_alh;
-
-		/* copier
-		 * {
-		 *  gtw_cfg
-		 *  {
-		 *     gtw_node_id;
-		 *     config_length;
-		 *     config_data
-		 *     {
-		 *	   count;
-		 *	  {
-		 *	     node_id;  \\ normal gtw id
-		 *	     mask;
-		 *	  }  mapping[MAX_ALH_COUNT];
-		 *     }
-		 *   }
-		 * }
-		 */
```

Comment:
is this... description not valid any more? I'm not entirely sure what it meant, but some documentation is often better than no documentation